### ISSUE TITLE
Dash to dock, "Installing Gnome Extensions" instructions possibly overcomplicated

### ISSUE THREAD

**Issue by brs17** [system76/docs#97]
Currently instructions mention install `chrome-gnome-shell` which I've never had to install to get dash to dock working.

Normally I just follow the instructions as guided on the extensions.gnome.org page: `Click here to install browser extension` 

![screenshot from 2018-07-02 08-48-09](https://user-images.githubusercontent.com/7460774/42171828-912577b0-7dd7-11e8-8192-be0ca552d005.png)

and then do the affirmative on any other popups that appear. May we want to suggest as the way to install GNOME shell integration? Also I believe installing gnome-tweak-tool is not necessary to install dash-to-dock (and there is no further mention of it on this page).

@ahoneybun @kylecorry31 Any thoughts/opinions?

**Comment by kylecorry31** [comment]
I agree, we should remove both references.

**Comment by ahoneybun** [comment]
@brs17 @kylecorry31 your referring to this section correct?

### Installing GNOME Extensions
To get started with installing GNOME extensions, install the GNOME Tweak Tool and Chrome GNOME Shell using the following commands:
```shell
sudo apt install gnome-tweak-tool chrome-gnome-shell
```

Then open Firefox or Chromium (or Google Chrome) and install the GNOME shell integration tool if needed:
- [Firefox extension](https://addons.mozilla.org/en-US/firefox/addon/gnome-shell-integration/)
- [Chrome extension](https://chrome.google.com/webstore/detail/gnome-shell-integration/gphhapmejobijbbhgpjhcjognlahblep?hl=en)

Once the shell integration software is done, you can move on to installing the Dash To Dock extension.

**Comment by brs17** [comment]
yes we are

**Comment by brs17** [comment]
@ahoneybun I was not suggesting we just remove the section without replacing it with something. I was thinking we should have brief instructions/screenshots of installing gnome shell integration from that purple popup in my screenshot above.

**Comment by brs17** [comment]
I'm going to close this issue now, although perhaps we would still want to mention `gnome-tweak-tool`?

**Comment by BStudent** [comment]
LoL for sure mention **gnome tweak tool** and **dash-to-dock**: my preference would be to _"mention" that tweak and dash-to-dock are installed by default on PoP_.  Most people I know who don't use them do so because they don't know they exist.  The point is that hardcore Linux people have _opinions_ about tweak-n-dash, while people coming from Mac and Windows just expect them - and those are the majority of users you want to cater to because _they are the ones who buy your hardware_.   From that standpoint, user opinions should be weighted by whether they represent the people you're hoping to sell hardware to.  A Linux person who's just going to download PoP for free and not buy hardware may have an articulate and well-informed opinion that is worth zero in terms of funding your future paychecks.  The potential Mac / Windows convert who is totally baffled by the difference between Gnome and KDE but wants to buy that sweet oryx pro because the new macbooks are pathetic has an opinion worth three grand give or take. 
_Why?_
AFAIK, **PoP is the first Linux distro that's a credible alternative to Mac/PC by just being _better_, period.**  To see the implications, let's think about the current market for desktop Linux.  
Outside of the datacenter, the Linux people who might actually spend money on a System76 computer are mainly: 
1. systems /open-source programmers ("Linux for Linux sake"), 
2. people in _some_ segments of the science / engineering world, and 
3. "real" Black and White -hats 
- the fact that many of our colleagues and internet buddies fall into these categories creates the illusion that We Are Legion.  We are not: we're a tiny minority of the population.  

There are lots more Linux users, but they have little impact on s76 growth and success e.g.: 
- Users in emerging markets who use Linux on a ten year old computer, 
- recently most 12-year-olds with a 3-digit IQ (therefore no more than half of all 12-year-olds) are downloading and ~installing~ trying to install Kali Linux so thy can be a badass ~script kiddie~ hacker (some small fraction do not give up after finding it's harder than installing FarCry5).  
- Neither of those groups are likely to buy a box from s76 
- Actually, the 1st-world 12yo might be able to get mom and dad to shell out for an s76 box running the (hypothetical) _PoP - 1331 K4L1 H4XX Edition_, as long as it prominently features Steam pre-installed.
 
So what?  Here's what: you have an opportunity to break out of this tiny segment and go after a much bigger market.
For sake of argument, let's take as a "given" my claim that _Pop/s76 is the first family of Linux systems that can compete with Macs and High-end PCs strictly on merit_: _what are the implications?_
The market for Macs and high-end PCs is orders of magnitude larger than the market for non-datacenter Linux machines.  Capturing a small piece of it can multiply your sales, and while we geek types think bean-counting beneath us, bean-count is what makes the difference between hiring/promotions and layoffs.  Given this opportunity I'd suggest viewing UX from the perspective of the Mac / Windows convert.
So why am I going off on this tangent?  Because it's not a tangent, it's a perfect example:
Bottom line:  Mac and Windows people will want Tweak and Dash because it's what they are used to - and Windows / OSX have those rather-similar features because Apple and Microsoft spent billions of dollars in UX research, and both came to the same basic design - even though each would prefer to have a look that is distinct from the other.

Just a thing to think about ....

B.
